### ISSUE TITLE
disable user password change if come in with ldap auth

### ISSUE THREAD

**Issue by tb-killa** [netbox-community/netbox#3139]
### Environment
* Python version: 3.6
* NetBox version: 2.5.11

### Proposed Functionality
Removal or deactivation of the option to change the password in the user interface if authentication is performed via ldap.

### Use Case
If the authentication via django-auth-ldap and the ldap_config.py file is used, the possibility to change the password must not be given in parallel in the menu of the user profile.

This change reacts within Netbox (Raw passwords) and not via the ldap plugin, so that you can also log in with your changed password.

### Database Changes
None.

### External Dependencies
None.

**Comment by tb-killa** [comment]
Found another ticket with same wish:
https://code.djangoproject.com/ticket/17967
